### PR TITLE
feat: add crkbd/keymaps/armand1m

### PR DESCRIPTION
**Pull request by armand1m**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/crkbd/keymaps/armand1m/keymap.c [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+}
+
+void oled_render_logo(void) {
+    static const char PROGMEM crkbd_logo[] = {
+        0x80, 0x81, 0x82, 0x83, 0x84, 0x85, 0x86, 0x87, 0x88, 0x89, 0x8a, 0x8b, 0x8c, 0x8d, 0x8e, 0x8f, 0x90, 0x91, 0x92, 0x93, 0x94,
+        0xa0, 0xa1, 0xa2, 0xa3, 0xa4, 0xa5, 0xa6, 0xa7, 0xa8, 0xa9, 0xaa, 0xab, 0xac, 0xad, 0xae, 0xaf, 0xb0, 0xb1, 0xb2, 0xb3, 0xb4,
+        0xc0, 0xc1, 0xc2, 0xc3, 0xc4, 0xc5, 0xc6, 0xc7, 0xc8, 0xc9, 0xca, 0xcb, 0xcc, 0xcd, 0xce, 0xcf, 0xd0, 0xd1, 0xd2, 0xd3, 0xd4,
+        0};
+    oled_write_P(crkbd_logo, false);
+}
```
Comment (armand1m) [inline keyboards/crkbd/keymaps/armand1m/keymap.c:144]: remove this function as it is not currently used

```suggestion
```

**Review by fauxpark — APPROVED** [review]

**Comment by armand1m** [comment]
do you folks recommend any specific software so I can generate the keymap layer as a png to put in the docs?

**Comment by drashna** [comment]
You could use keyboard layout editor to create that and export as an image, IIRC.
However, said image would need to be hosted on imgur or the like, as we don't accept images in the repo, to keep it's file size down. 

Otherwise, this PR is ready to go. Though want to make sure if you wanted it merged yet, or not.

**Comment by armand1m** [comment]
@drashna I'm ok with merging this PR, I might open another one in a month or so with the image layouts and some other improvements. Thanks for the guidance!

**Comment by drashna** [comment]
Sounds good!

**Pull request merged**